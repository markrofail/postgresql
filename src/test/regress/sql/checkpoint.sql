-- CHECKPOINT

CHECKPOINT;
CHECKPOINT (SPREAD);

CHECKPOINT (SPREAD 2);
CHECKPOINT (NONEXISTENT);
