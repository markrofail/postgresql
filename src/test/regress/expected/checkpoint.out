-- CHECKPOINT
CHECKPOINT;
CHECKPOINT (SPREAD);
CHECKPOINT (SPREAD 2);
ERROR:  spread requires a Boolean value
CHECKPOINT (NONEXISTENT);
ERROR:  unrecognized CHECKPOINT option "nonexistent"
LINE 1: CHECKPOINT (NONEXISTENT);
                    ^
